aw_watcher_terminal/main.py

```diff
@@ -66,7 +66,8 @@ def init_client():
 
     # Create Bucket if not already existing
     bucket_id = "{}_{}".format(config['bucket_name'], client.hostname)
-    client.create_bucket(bucket_id, event_type=config['event_type'])
+    client.create_bucket(bucket_id, event_type=config['event_type'],
```

Comment:
Please don't have the event_type, bucketname or client_id configurable, please hardcode them.
There's no point in configuring them, will likely just break things from people modifying the config file and doesn't know what they are doing.

Reply:
The intention of config.py is to provide global configurations/variables for the other scripts, not to make all these editable by the user (even though I'll likely add it for a subset of the configuration variables in the future).

I think event_type, bucketname and client_id are only used in main.py so I can move them there if you want.

The config file was also updated a bit, you can view the current version [here](https://github.com/Otto-AA/aw-watcher-terminal/blob/7c5db14eb54aa45b6e2fb91b091b22fb423e610f/aw_watcher_terminal/config.py).

If you have recommendations about that, please open another issue. To be honest I am not so happy with the current format of it, but it's good enough for now imo.